src/platform/packages/shared/response-ops/alerts-table/components/alerts_table.test.tsx

```diff
@@ -780,86 +760,6 @@ describe('AlertsTable', () => {
     });
   });
 
-  describe('Flyout', () => {
```

Comment:
I moved these is in [alerts_data_grid.test.tsx](https://github.com/elastic/kibana/pull/233697/files#diff-e95f92e213daf3ed89ff46c52a14f8f006edc6880183315231521c51ae560100), since the expanded alert view rendering logic is in that component